ajax/libs/instagram-lite/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "instagram-lite",
+  "version": "2.0.0",
+  "filename": "instagramLite.min.js",
+  "description": "A simple, lightweight jQuery plugin used to display a user's Instagram photos.",
+  "keywords": [
+    "instagram",
+    "photos",
+    "api",
+    "feed"
+  ],
+  "license": "MIT",
+  "author": "Michael Lynch <mikethomaslynch@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/michael-lynch/instagram-lite"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/michael-lynch/instagram-lite.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "instragramLite*"
```

Comment:
@sauloaguiar Would you modify from `instragramLite` to `instagramLite`?
There should remove r after the t. (inst"r"agramLite )

And would you help remove one space between `2.0.0` & `w/` in the commit message? as the comment https://github.com/cdnjs/cdnjs/pull/11583#issuecomment-318040279

Sorry that I did not notice. If you have any question, please let me know.
Thank you for your help.